qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -97,6 +96,10 @@ def __init__(
         self._full_sampling = full_sampling
         self._clifford_utils = CliffordUtils()
 
+        # Evaluated in the execution stage. Values are copied to experiment data metadata.
+        self._gate_error_ratio = None
```

Comment:
I think these can be removed, we shouldn't have a duplicate variable storing what is also an experiment option.